lib/circactivator/checkupdater.rb

```diff
@@ -44,11 +44,33 @@ def url
     end
 
     def fetch
-      response = HTTParty.get(url + '?query_broker=1', headers: http_headers, verify: false)
-      raise_exceptions!(response)
+      response = call_circonus(:get, url + '?query_broker=1')
       @check_bundle = JSON.load(response.body)
     end
 
+    def update
+      response = call_circonus(:put, url, body: payload_hash.to_json)
+    end
+
+    def call_circonus(method, url, options={})
+      options[:headers] = http_headers
+      options[:verify]  = CircActivator::Config.circonus['verify'] || true
```

Comment:
Damn it, I just made this mistake in something else two weeks ago and fixed it.  Apparently, I never learn.

Rather than adding another method, I'd rather change this to:

options[:verify] = CircActivator::Config.circonus.fetch('verify', false)

What do you think?